tests/integration/records/multi_db/test_records_table2table.py

```diff
@@ -96,18 +95,15 @@ def move_and_verify(self, source_dbname, target_dbname, variant_used_internally)
 
 def create_test_func(source_name, target_name):
     def source2target(self):
-        self.move_and_verify(source_name, target_name,
-                             variant_used_internally='vertica')
+        self.move_and_verify(source_name, target_name)
     return source2target
 
 
 if __name__ == '__main__':
+    set_stream_logging()
+
     for source in DB_TYPES:
         for target in DB_TYPES:
-            variant_used_internally = 'bluelabs'
-            if 'vertica' in [source, target]:
-                variant_used_internally = 'vertica'
-
```

Comment:
You'll notice this never gets passed in 😆 

It's now the responsibility of `table_validator.py` to figure out which variant is exported from the source database and imported to the target database.  It's the responsibility of `test_records_table2table.py` to tell which databases are being moved between.